client/components/marketing-survey/cancel-auto-renewal-form/index.jsx

```diff
@@ -0,0 +1,148 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize, moment } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Dialog from 'components/dialog';
+import FormSectionHeading from 'components/forms/form-section-heading';
+import FormButton from 'components/forms/form-button';
+import FormButtonsBar from 'components/forms/form-buttons-bar';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import FormTextInput from 'components/forms/form-text-input';
+import { submitSurvey } from 'lib/upgrades/actions';
+import enrichedSurveyData from 'components/marketing-survey/cancel-purchase-form/enriched-survey-data';
+import './style.scss';
+
+const OTHER_FEEDBACK = 'other-feedback';
+
+class CancelAutoRenewalForm extends Component {
+	static propTypes = {
+		purchase: PropTypes.object.isRequired,
+		selectedSite: PropTypes.object.isRequired,
+		isVisible: PropTypes.bool,
+		onClose: PropTypes.func.isRequired,
+		translate: PropTypes.func.isRequired,
+	};
+
+	state = {
+		response: '',
+		feedback: '',
+	};
+
+	radioButtons = {};
+
+	constructor( props ) {
+		super( props );
+
+		const { translate } = props;
+
+		this.radioButtons = [
+			[ 'take-a-break', translate( "Yes, I'm just taking a break for now." ) ],
+			[ 'manual-renew', translate( 'Yes, I want to renew manually.' ) ],
+			[ 'stop-renew', translate( 'No, I don’t have any plans to renew it.' ) ],
+			[ OTHER_FEEDBACK, translate( 'Another reason' ) ],
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Another reason:' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/enhance/auto-renewal-survey)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).